core/modules/layout/css/layout.admin.css

```diff
@@ -173,7 +180,7 @@
 .layout-editor-region-title .dropbutton-wrapper {
   float: left; /* LTR */
   width: 140px;
-  margin: 4px 0 10px
+  margin: 4px 00;
```

Comment:
??

Reply:
oops, fixed! (I was trying to add the `;`)